docs/gallery/interactive/pyfar_interactive_plots.ipynb

```diff
@@ -0,0 +1,79 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Interactive plots\n",
+    "\n",
+    "pyfar provides convenient **interactive keyboard shortcuts**.\n",
+    "These allow switching plots, zooming in and out, moving along the x and y-axis, and zooming and moving the range of colormaps.\n",
+    "\n",
+    "To do this, you need to use an interactive matplotlib backend.\n",
+    "For jupyter notebooks one can for example use the matplotlib magic `%matplotlib ipympl`.\n",
+    "\n",
+    "The available keyboard shortcuts can be queried using\n",
+    "```python\n",
+    "print(pf.plot.shortcuts())\n",
```

Comment:
```suggestion
    "pf.plot.shortcuts()\n",
```
and we can convert it to an executed cell.

Reply:
If I remember correctly, multiple people preferred it this way and linking to the table in the docs. Thus I will keep it this way.

Reply:
If you refer to the table in the docs, you don't need the command. In the Pyfar intro we also execute it. Maybe we can do it in a similar way here.

Reply:
Running it makes the nb very bloated. With this, all options are included in the nb.